tests/testthat/helper.R

```diff
@@ -0,0 +1,4 @@
+check_suggests <- function() {
+  skip_if_not_installed("rvest")
+  skip_if_not_installed("xml2")
+}
```

Comment:
Defining this often-used helper in `tests/testthat/helper.R` is the main move here, which allows us to remove it from many test files.